bridges/snowbridge/primitives/core/src/location.rs

```diff
@@ -100,6 +106,18 @@ impl DescribeLocation for DescribeTokenTerminal {
 	}
 }
 
+/// Conversion implementation which converts from a `[u8; 32]`-based `AccountId` into a
+/// `Location` consisting solely of a `AccountId32` junction with a fixed value for its
+/// network and the `AccountId`'s `[u8; 32]` datum for the `id`.
+pub struct AccountToLocation<AccountId>(PhantomData<AccountId>);
+impl<AccountId: Clone + Into<[u8; 32]> + Clone, Location: core::convert::From<Junction>>
+	Convert<AccountId, Location> for AccountToLocation<AccountId>
+{
+	fn convert(who: AccountId) -> Location {
+		AccountId32 { network: None, id: who.clone().into() }.into()
+	}
+}
+
```

Comment:
Is this somewhat duplicated with `xcm-builder`

https://github.com/paritytech/polkadot-sdk/blob/6b980646920a1268b491b3b37c266aa486d66eb2/polkadot/xcm/xcm-builder/src/location_conversion.rs#L352-L362

and I'd suggest this is provided as a runtime config, without ignoring the network